sentry-spring/src/test/kotlin/io/sentry/spring/reactive/SentryReactiveWebRequestProcessorTest.kt

```diff
@@ -0,0 +1,105 @@
+package io.sentry.spring.reactive
+
+import io.sentry.SentryEvent
+import io.sentry.SentryOptions
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFalse
+import kotlin.test.assertNull
+import kotlin.test.assertTrue
+import org.springframework.http.MediaType
+import org.springframework.mock.http.server.reactive.MockServerHttpRequest
+
+class SentryReactiveWebRequestProcessorTest {
+
+    @Test
+    fun `attaches basic information from HTTP request to SentryEvent`() {
+        val request = MockServerHttpRequest.get("http://example.com?param1=xyz")
+            .header("some-header", "some-header value")
+            .accept(MediaType.APPLICATION_JSON)
+            .build()
+        val eventProcessor = SentryReactiveWebRequestProcessor(request, SentryOptions())
```

Comment:
can we create a `getSut` here that creates a new instance of `SentryReactiveWebRequestProcessor` on each test? an `SentryOptions` param too, with default value of `SentryOptions()`, so you dont need to create a instance of `SentryReactiveWebRequestProcessor` and `SentryOptions` in some specific tests, see an example:

https://github.com/getsentry/sentry-java/blob/5ebd49cbfd17eb66175e9247db6d27e4df84a56b/sentry-android-timber/src/test/java/io/sentry/android/timber/SentryTimberTreeTest.kt

Reply:
Thanks for the suggestion @marandaneto